src/GafferImage/Mix.cpp

```diff
@@ -186,6 +230,21 @@ void Mix::hashChannelData( const GafferImage::ImagePlug *output, const Gaffer::C
 {
 	ImageProcessor::hashChannelData( output, context, h );
 
+	float mix = mixPlug()->getValue();
```

Comment:
I'm trying to get into the habit of making everything possible const, so the compiler will tell me when I make mistakes like `if( mix = 0.0f )`, and so there's less to think about when following the code flow. Would be nice to see this be const (and the others), but no big deal...